### PR TITLE
add glossary defs client-side, server-side, client-server

### PR DESCRIPTION
**Pull request by genwhittTTD**
add glossary defs client-side, server-side, client-server.

### CODE REVIEW

File: docs/ref-info/glossary-uid.md [hunk 1]
```diff
@@ -151,6 +154,15 @@ import Link from '@docusaurus/Link';
...
 <dd>For details, see <a href="/docs/getting-started/gs-credentials">UID2 Credentials</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-client-server">Client-server integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the <a href="#gl-client-side">client side</a>, entirely on the <a href="#gl-server-side">server side</a>, or partially on the client side and partially on the server side (client-server).</dd>
+<dd>For example, in a client-server integration for a publisher, the UID2 token is generated on the server side and refreshed on the client side.</dd>
+
+<dt><MdxJumpAnchor id="gl-client-side">Client-side integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the client side, entirely on the <a href="#gl-server-side">server side</a>, or partially on the client side and partially on the server side (<a href="#gl-client-server">client-server</a>).</dd>
+<dd>in a client-side integration, all activities relating to UID2, such as generating and refreshing UID3 tokens, are performed on the client side.</dd>
```
Comment (sunnywu) [inline docs/ref-info/glossary-uid.md:163]: capital I for _in a client-side integration_
and NOT _UID3 tokens_ 🤣

File: docs/ref-info/glossary-uid.md [hunk 2]
```diff
@@ -151,6 +154,15 @@ import Link from '@docusaurus/Link';
 <dd>Each UID2 <a href="/docs/intro#participants">participant</a> using a server-side implementation has an API key (client key) and also a secret value associated with the key, called the client secret (API secret). The client secret is known only to the participant and the UID2 service.</dd>
 <dd>For details, see <a href="/docs/getting-started/gs-credentials">UID2 Credentials</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-client-server">Client-server integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the <a href="#gl-client-side">client side</a>, entirely on the <a href="#gl-server-side">server side</a>, or partially on the client side and partially on the server side (client-server).</dd>
+<dd>For example, in a client-server integration for a publisher, the UID2 token is generated on the server side and refreshed on the client side.</dd>
```
Comment (sunnywu) [inline docs/ref-info/glossary-uid.md:159]: should we provide example, e.g. link to docs/guides/integration-mobile-client-server and /docs/guides/integration-prebid-server-side ?

File: docs/ref-info/glossary-uid.md [hunk 3]
```diff
@@ -151,6 +154,15 @@ import Link from '@docusaurus/Link';
...
+
+<dt><MdxJumpAnchor id="gl-client-side">Client-side integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the client side, entirely on the <a href="#gl-server-side">server side</a>, or partially on the client side and partially on the server side (<a href="#gl-client-server">client-server</a>).</dd>
+<dd>in a client-side integration, all activities relating to UID2, such as generating and refreshing UID3 tokens, are performed on the client side.</dd>
+<dd>For example, in a client-side integration, advertisers generate UID2 tokens on the client side for tracking pixels, and publishers generate UID2 tokens on the client side for bidstream use, as well as refreshing the tokens.</dd>
```
Comment (sunnywu) [inline docs/ref-info/glossary-uid.md:164]: same suggestion: maybe link to docs/guides/integration-mobile-client-side and docs/guides/publisher-client-side docs/guides/integration-prebid-client-side ?

File: docs/ref-info/glossary-uid.md [hunk 4]
```diff
@@ -334,6 +346,11 @@ import Link from '@docusaurus/Link';
 <dd>For details, see <a href="https://blog.google/products/admanager/new-ways-for-publishers-to-manage-first-party-data/">Share secure signals with your trusted partners</a> (second section) and <a href="https://support.google.com/admanager/answer/10488752?hl=en">Share secure signals with bidders</a>, both from Google.</dd>
 <dd>For details about UID2 support of the Google secure signals feature, see <a href="../guides/google-ss-integration">Google Ad Manager Secure Signals Integration Guide</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-server-side">Server-side integration</MdxJumpAnchor></dt>
+<dd>UID2 integrations can be entirely on the <a href="#gl-client-side">client side</a>, entirely on the server side, or partially on the client side and partially on the server side (<a href="#gl-client-server">client-server</a>).</dd>
+<dd>In a server-side integration, all activities relating to UID2, such as generating and refreshing UID2 tokens, are performed on the server.</dd>
+<dd>For example, in a server-side integration, advertisers generate raw UID2s on the server side to be delivered for audience targeting, and publishers generate UID2 tokens on the server side for bidstream use.</dd>
```
Comment (sunnywu) [inline docs/ref-info/glossary-uid.md:352]: maybe link to docs/guides/custom-publisher-integration ?

**Comment by sunnywu** [comment]
Maybe in a new ticket - should we rename
https://unifiedid.com/docs/guides/integration-javascript-server-side
https://unifiedid.com/docs/guides/integration-prebid-server-side
as client-server guide? technically it is asking pub to generate tokens on server side and then pass the identity to the js sds/prebid.js  in the client side to auto refresh so it is client-server by your definition?

File: docs/ref-info/glossary-uid.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/ref-info/glossary-uid.md:163]: Ouchy! TYSM @sunnywu . Fixed.

File: docs/ref-info/glossary-uid.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/ref-info/glossary-uid.md:159]: Done, thx. See my query to you in Slack.

File: docs/ref-info/glossary-uid.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/ref-info/glossary-uid.md:164]: Done. Thx much.

File: docs/ref-info/glossary-uid.md (hunk 4 above)
Comment (genwhittTTD) [reply docs/ref-info/glossary-uid.md:352]: Done, thx.

**Pull request merged**